source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst

```diff
@@ -59,312 +62,14 @@ deployments are best suited for evaluation and initial development environments.
    For deployments that *require* using network-attached storage, use
    NFSv4 for best results.
 
-1) Download the MinIO Server
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. cond:: linux
-
-   .. include:: /includes/linux/common-installation.rst
-      :start-after: start-install-minio-binary-desc
-      :end-before: end-install-minio-binary-desc
-
-.. cond:: macos
-
-   .. include:: /includes/macos/common-installation.rst
-      :start-after: start-install-minio-binary-desc
-      :end-before: end-install-minio-binary-desc
-
-2) Download and Run MinIO Server
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. cond:: linux
-
-   .. include:: /includes/linux/common-installation.rst
-      :start-after: start-run-minio-binary-desc
-      :end-before: end-run-minio-binary-desc
-
-.. cond:: macos
-
-   .. include:: /includes/macos/common-installation.rst
-      :start-after: start-run-minio-binary-desc
-      :end-before: end-run-minio-binary-desc
-
-3) Add TLS Certificates
```

Comment:
Intentionally dropped, as the dedicated TLS tutorial page can provide better instructions. If we want to go this route, it should be as part of a polish pass where the deploy tutorials cover *all* possible setups, including TLS, KES, etc. That requires significantly more effort and care than we have bandwidth for today.